libs/data-mapper/src/lib/utils/Layout.Utils.ts

```diff
@@ -1,67 +1,55 @@
 import { functionNodeCardSize, schemaNodeCardDefaultWidth, schemaNodeCardHeight } from '../constants/NodeConstants';
+import { targetPrefix } from '../constants/ReactFlowConstants';
 import type { SchemaNodeDictionary, SchemaNodeExtended } from '../models';
 import type { ConnectionDictionary } from '../models/Connection';
 import type { FunctionDictionary } from '../models/Function';
 import { generateInputHandleId, isConnectionUnit } from './Connection.Utils';
 import { isFunctionData } from './Function.Utils';
+import { LogCategory, LogService } from './Logging.Utils';
```

Comment:
BEGIN a bunch of deleting ELK stuff while adapting some of it (for example, the ElkNode / elk graph structure) to the new use case